wallet/wallet.c

```diff
@@ -3377,15 +3377,15 @@ wallet_payment_list(const tal_t *ctx,
 				      " WHERE (payment_hash = ? OR ?) AND (status = ? OR ?)"
 				      " ORDER BY id;"));
 
-        db_bind_sha256(stmt, 0, payment_hash == NULL ? NULL : payment_hash);
+	db_bind_sha256(stmt, 0, payment_hash);
```

Comment:
This is exactly the right place to disambiguate whether the value is `NULL` or not, no need to push it down the stack.

Reply:
I agree, is more clear to read also from a user viewpoint.

Thanks @cdecker